eclass/gnome2.eclass

```diff
@@ -233,40 +146,31 @@ gnome2_src_install() {
 	#
 	# if this is not present, scrollkeeper-update may segfault and
 	# create bogus directories in /var/lib/
-	if has ${EAPI} 5; then
-		dodir "${sk_tmp_dir}" || die "dodir failed"
-		emake DESTDIR="${D}" "scrollkeeper_localstate_dir=${ED}${sk_tmp_dir} " "$@" install
-	else
-		default
-	fi
+	default
 
 	unset GCONF_DISABLE_MAKEFILE_SCHEMA_INSTALL
 
 	# Handle documentation as 'default' for eapi5, bug #373131
 	# Since eapi6 this is handled by default on its own plus MAINTAINERS and HACKING
 	# files that are really common in gnome packages (bug #573390)
-	if has ${EAPI} 5; then
-		einstalldocs
-	else
-		local d
-		for d in HACKING MAINTAINERS; do
-			[[ -s "${d}" ]] && dodoc "${d}"
-		done
-	fi
+	local d
+	for d in HACKING MAINTAINERS; do
+		[[ -s ${d} ]] && dodoc "${d}"
+	done
 
 	# Do not keep /var/lib/scrollkeeper because:
 	# 1. The scrollkeeper database is regenerated at pkg_postinst()
 	# 2. ${ED}/var/lib/scrollkeeper contains only indexes for the current pkg
 	#    thus it makes no sense if pkg_postinst ISN'T run for some reason.
-	rm -rf "${ED}${sk_tmp_dir}"
-	rmdir "${ED}/var/lib" 2>/dev/null
-	rmdir "${ED}/var" 2>/dev/null
+	rm -rf "${ED}${sk_tmp_dir}" || die
+	rmdir "${ED}/var/lib" 2>/dev/null || die
+	rmdir "${ED}/var" 2>/dev/null || die
```

Comment:
Hm, I suppose there wasn't a `die` because these dirs may not exist. The stderr redirection points to that, too.